### ISSUE TITLE
Issue when building with cmake

### ISSUE THREAD

**Issue by Alex533704** [vitasdk/samples#77]
Hello, i try to build the hello-world sample using cmake (GUI)
Building the sample using cmake  without GUI works succesfully, but when i try to build using cmake  GUI,  it fails.

At the beginning cmake asks me to choose a generator for the project , wich generator should i choose ?

**Comment by isage** [comment]
Obviously, the one, for which you have build tools available. Typically it's `Unix Makefiles`

**Comment by Alex533704** [comment]
Hello, when i choose "Unix Makefiles"  as build  programm i have this error: 
**CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.**

**Comment by isage** [comment]
You're using windows tool for toolchain installed in mingw/msys.
Use MinGW makefiles or msys makefiles generator.
Or, just don't use gui, there's zero reasons to use it. Use `ccmake` (console gui) instead

**Comment by Alex533704** [comment]
Hello, i successfully created  a **Makefile** with **cmake (GUI)**  and i used as generator "MinGW makefiles".

But when tried t build using "Make" command  a received a  weird  message from Console and there wasn't any compilation at all.



This is the message i received in Console after typing **"make"**        -->  **(c) microsoft corporation. all rights reserved**

**Comment by isage** [comment]
If you're using MinGW makefiles use `mingw32-make`. Read the cmake docs.
